ipn/local.go

```diff
@@ -226,12 +231,16 @@ func (b *LocalBackend) setClientStatus(st controlclient.Status) {
 		b.mu.Unlock()
 
 		b.send(Notify{NetMap: st.NetMap})
-		// There is nothing to update if the map hasn't changed.
-		if changed {
-			b.updateFilter(st.NetMap)
+
+		b.updateFilter(st.NetMap)
+		b.e.SetNetworkMap(st.NetMap)
+
+		if !dnsMapsEqual(st.NetMap, oldNetMap) {
 			b.updateDNSMap(st.NetMap)
-			b.e.SetNetworkMap(st.NetMap)
+		} else {
+			b.logf("dns map: (not changed)")
```

Comment:
I'd rather you only logged if it did change. (which you don't currently).

Not changing is the normal case so we can save some log spam.